### ISSUE TITLE
Add jar support

### ISSUE THREAD

**Issue by biom4st3r** [mnxn/vscode-jvm-bytecode-viewer#1]
I was wondering if you're willing to add support for reading .class files from jars as most of the bytecode i need to work with are in my dependencies

**Comment by mnxn** [comment]
I will see what I can do.

javap supports reading classes from jar files:

```
Each class to be shown may be specified by a filename, a URL, or by its fully
qualified class name. Examples:
   path/to/MyClass.class
   jar:file:///path/to/MyJar.jar!/mypkg/MyClass.class
   java.lang.Object
```

so what needs to be implemented is a way to explore jar files from VSCode. The code can be adapted from https://github.com/stef-levesque/vscode-zipexplorer.

**Comment by biom4st3r** [comment]
i think jszip can open a jar. i'm not familiar with ts or js, but i'm trying it out

**Comment by mnxn** [comment]
It was more work than I expected, but this is now implemented in v0.3.0.

See https://github.com/mnxn/vscode-jvm-bytecode-viewer#usage for details.

**Comment by biom4st3r** [comment]
Thank you so much!
